discovery/db_discovery.go

```diff
@@ -93,7 +93,7 @@ func (dbo *DBOrchestratorPoolCache) GetOrchestrators(numOrchestrators int) ([]*n
 }
 
 func (dbo *DBOrchestratorPoolCache) Size() int {
-	orchs, err := dbo.node.Database.SelectOrchs()
+	orchs, err := dbo.node.Database.SelectOrchs(nil)
```

Comment:
We should pass in a filter with the max price here

Reply:
Right, was thinking that after the discussion earlier but wasn't sure. 
fixed in https://github.com/livepeer/go-livepeer/pull/939/commits/91536e4e33b71ae4cc5bcf1078ddda29ec399bff